From e809ae77e68e4c91273583579ffcaa103c334cf6 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Fri, 6 Nov 2020 12:52:59 -0800
Subject: [PATCH 19/34] SystemUI: Disable blur wakeup animation

This animation looks distracting and prevents the user from seeing
anything during the unlock animation.

Change-Id: I2c8ff9e6a9ec361595bc725d971a51e91acc13a2
---
 .../systemui/statusbar/NotificationShadeDepthController.kt      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/NotificationShadeDepthController.kt b/packages/SystemUI/src/com/android/systemui/statusbar/NotificationShadeDepthController.kt
index 20ab4057..5bec6854 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/NotificationShadeDepthController.kt
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/NotificationShadeDepthController.kt
@@ -65,7 +65,7 @@ class NotificationShadeDepthController @Inject constructor(
     dumpManager: DumpManager
 ) : PanelExpansionListener, Dumpable {
     companion object {
-        private const val WAKE_UP_ANIMATION_ENABLED = true
+        private const val WAKE_UP_ANIMATION_ENABLED = false
         private const val VELOCITY_SCALE = 100f
         private const val MAX_VELOCITY = 3000f
         private const val MIN_VELOCITY = -MAX_VELOCITY
-- 
2.30.1

